functions.sh

```diff
@@ -21,11 +21,23 @@ function getRepositoryTag() {
 function logInfoMessage() {
     MESSAGE="$1"
     CURRENT_DATE=`date "+%D: %T"`
-    echo "[$CURRENT_DATE] [INFO] $MESSAGE"
+    COLOUR_GREEN="\e[32m[INFO]\e[0m"
```

Comment:
Let's make these color codes global variable instead of defining them in a function

Reply:
Something like COLOR_GREEN_START COLOR_GREEN_END

Reply:
sir, made the changes please check.